UnityProject/Assets/Scripts/Systems/Electricity/Electrical processes/ElectricalPool.cs

```diff
@@ -146,4 +146,21 @@ public static ElectricalSynchronisation.QEntry GetQEntry()
 			return (new ElectricalSynchronisation.QEntry());
 		}
 	}
+
+	public static List<SupplyBool> PooledSupplyBool
```

Comment:
Codacy found an issue: [Change the visibility of 'PooledSupplyBool' or make it 'const' or 'readonly'.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6381045)